src/clients/KubectlClient.ts

```diff
@@ -101,10 +100,71 @@ export class KubectlClient implements IClient {
         }
     }
 
+    public async getContainersList(podName: string): Promise<string[]> {
+        if (!podName) {
+            this._logger.error(TelemetryEvent.KubectlClient_GetPodNameError, new Error(`Pod name is null`));
+            return null;
+        }
+        try {
+            const knownSideCars: string[] = ['linkerd-proxy', 'linkerd-init', 'istio-proxy', 'darpd', 'jaeger-agent', 'nginx-proxy'];
+            const kubeconfigPath: string = await this._accountContextManager.getKubeconfigPathAsync();
```

Comment:
sure, will add comments, these are known common one's but they are many so that's why introduced quick pick one so that we don't have to maintain this list. (future it might grow) :).

Reply:
I think I've asked this question before so sorry if there's an answer somewhere but...

Is it possible that someone might _want_ to use Bridge to debug one of these known sidecars? E.g. an Istio developer might be trying to diagnose an issue with their own software, and be wondering why their container is not appearing in the list?

If that's a possibility, could we maybe use defaults rather than completely excluding known sidecars?